code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -176,21 +221,32 @@
 		return
 
 	var/staggerslow_stacks = 2
+	var/paralyzesecs = 1 SECONDS
```

Comment:
camel case

Reply:
i dont know what that means, but except the stealth flag thats the original behavior.